README.md

```diff
@@ -6,12 +6,118 @@ output:
+
+-----
+
+#### Generated applications code structure
+
+- Regardless files contents, both packages generated applications share the same first level folder structure:
+
+<center><img src="./vignettes/figures/App_Structure.png"></center>
+
+- After that, there are differences in next level structure as follow:
+  - **program** folder, that contains application main logic, periscope2 has:
+    - a dedicated UI R script for each layout part to control it
+    - **config**  - folder for storing application different configuration files (it will evolve more in next versions)
+      - Initially contains announcements module configuration file
+    - **modules**  - folder for application shiny modules R scripts (it will evolve more in next versions)
+    <center><img src="./vignettes/figures/Program_Folder.png"></center>
+  - ***www*** folder that contains the application JS, CSS and theme customization files
+    - Although generated applications by both frameworks will have the same folder structure, periscope2 generated applications will have two additional files **css/custom.css** and **js/custom.js** to have application custom JS and CSS code
+      - Note that both files are already linked to generated app and ready to be used immediately by user
+
+----
+
+#### Packages modules
+
+- Both packages share some modules (for now) as downloadable file, downloadable plot and downloadable table
+- However, periscope2 has more modules (more to come with each new version) as Announcement module
+- Also, there are modules that exist in both packages but behave a little differently than each other (i.e logger and Reset Application) modules
+- Check both packages for each package modules details
+
+----
+
+#### Old Shiny module style support
+
+While [periscope](https://cran.r-project.org/web/packages/periscope/index.html) modules can be called via shiny module old style (via `callModule` method) or using new shiny module style.  periscope2 modules can only be called using new shiny module style. Check [migrating to module server](https://shiny.posit.co/r/articles/improve/modules/#migrating-from-callmodule-to-moduleserver) article for more related technical info
+
+----
+
+#### fresh package Styling Support
+[fresh](https://dreamrs.github.io/fresh/) package is used in both packages to style generated applications different parts via configurations rather than code via **'www/periscope_style.yaml'** file.
+
+However, [shiny dashboard variables](https://dreamrs.github.io/fresh/articles/vars-shinydashboard.html) are used for styling [periscope](https://cran.r-project.org/web/packages/periscope/index.html) generated applications. While [bs4Dash related variables](https://dreamrs.github.io/fresh/articles/vars-bs4dash.html) are used for styling periscope2 generated applications but not via code only via **'www/periscope_style.yaml'** configuration file (hence the file key names are user friendly in both packages)
```

Comment:
I'd work this sentence around the other way around and drop the periscope link and shinydashboard link:
periscope2 uses bs4dash related variables for styling in code and via the ... configuration file.  The file key names use shiny dashboard variables for user friendly styling in both packages.

Reply:
done